snippets/snippets.json

```diff
@@ -507,6 +507,48 @@
         "description": "For SEPTIC <= v2.81"
     },
     "Mvr": {
+        "prefix": "mvr",
+        "body": [
+            "  Mvr:           ${1:MvrName}",
+            "         Text1=  \"${4}\"",
+            "         Text2=  \"\"",
+            "          Mode=  STOPPED",
+            "          Auto=  OFF",
+            "       PlotMax=  100",
+            "       PlotMin=  0",
+            "      PlotSpan=  -1",
+            "          Nfix=  ${2:1}",
+            "        MaxChg=  -1",
+            "          Unit=  \"${3}\"",
+            "          Meas=  0",
+            "       GrpMask=  0000000000000000000000000000001",
+            "       GrpType=  0000000000000000000000000000000",
+            "          Span=  10",
+            "        HighOn=  1e+10",
+            "         LowOn=  -1e+10",
+            "  ProcessValue=  0",
+            "         IvOff=  0",
+            "         MaxUp=  10",
+            "         MaxDn=  -10",
+            "     MovePnlty=  1",
+            "         IvRoc=  -1",
+            "        IvPrio=  99",
+            "          Fulf=  1",
+            "   FulfReScale=  0",
+            "         Price=  0",
+            "      Blocking=  6    2     4     7    11    16    22 ",
+            "MeasValidation=  OFF",
+            " MeasHighLimit=  1e+10",
+            "  MeasLowLimit=  -1e+10",
+            "        LockHL=  OFF",
+            "        LockIV=  OFF",
+            "        LockLL=  OFF",
+            "UseFactorWeight=  0\r",
+            "        PVPred=  OFF"
+        ],
+        "description": "For SEPTIC >= v2.89"
```

Comment:
Description kan gjerne stå tom - brukes i utgangspunktet bare for eldre varianter av snippets.